commit e86e45fdb64ca071b722b199f4bdac941b673ec8
Author: CODeRUS <coderusinbox@gmail.com>
Date:   Mon Aug 18 23:53:28 2014 +0600

    modified

diff --git a/compositor.qml b/compositor.qml
index 3159661..beb9427 100644
--- a/usr/share/lipstick-jolla-home-qt5/qml/compositor.qml
+++ b/usr/share/lipstick-jolla-home-qt5/qml/compositor.qml
@@ -1087,7 +1087,7 @@ Compositor {
     onDisplayOff: { PeekFilter.cancelGesture(); hideDisplayOffRectangle() }
     onDisplayOn: hideDisplayOffRectangle()
 
-    PowerMenuDialog {}
+    PowerMenuDialog { parent: layersParent; z: 4 }
 
     UnresponsiveApplicationDialog {
         id: unresponsiveApplicationDialog
